src/forms/renderers/MultiLineSecret.tsx

```diff
@@ -7,17 +7,19 @@ import { withJsonFormsControlProps } from '@jsonforms/react';
 import { Box, Button, Stack } from '@mui/material';
 import { useCallback, useState } from 'react';
 import { DropzoneDialog } from 'react-mui-dropzone';
-
-export const MULTI_LINE_SECRET = 'multiLineSecret';
+import { Formats, Options } from 'types/jsonforms';
 
 export const multiLineSecretTester: RankedTester = rankWith(
-    999,
-    and(optionIs('format', MULTI_LINE_SECRET))
+    10,
+    and(
+        optionIs(Options.format, Formats.password),
+        optionIs(Options.multi, true)
+    )
```

Comment:
Better way to test for this as using a custom format caused issues trying to get the text input to be `password`.